spark/src/test/scala/org/apache/comet/exec/CometColumnarShuffleSuite.scala

```diff
@@ -925,193 +831,64 @@ abstract class CometShuffleSuiteBase extends CometTestBase with AdaptiveSparkPla
     }
   }
 
-  test("hash shuffle: Comet shuffle") {
-    // Disable CometExec to explicit test Comet Arrow shuffle path
-    Seq(true, false).foreach { execEnabled =>
-      withSQLConf(
-        CometConf.COMET_EXEC_ENABLED.key -> execEnabled.toString,
-        CometConf.COMET_EXEC_SHUFFLE_ENABLED.key -> "true",
-        CometConf.COMET_COLUMNAR_SHUFFLE_ENABLED.key -> (!execEnabled).toString) {
-        withParquetTable((0 until 5).map(i => (i, (i + 1).toLong)), "tbl") {
-          val df = sql("SELECT * FROM tbl").sortWithinPartitions($"_1".desc)
-          val shuffled1 = df.repartition(10, $"_1")
-
-          // If Comet execution is disabled, `Sort` operator is Spark operator
-          // and jvm arrow shuffle is applied.
-          checkCometExchange(shuffled1, 1, execEnabled)
-          checkSparkAnswer(shuffled1)
-
-          val shuffled2 = df.repartition(10, $"_1", $"_2")
-
-          checkCometExchange(shuffled2, 1, execEnabled)
-          checkSparkAnswer(shuffled2)
-
-          val shuffled3 = df.repartition(10, $"_2", $"_1")
-
-          checkCometExchange(shuffled3, 1, execEnabled)
-          checkSparkAnswer(shuffled3)
-        }
-      }
-    }
-  }
-
-  test("Comet shuffle: different data type") {
```

Comment:
We have this which tests both columnar and native shuffle, and "Comet columnar shuffle shuffle: different data type" below which only test columnar shuffle. This PR moves the columnar shuffle from this test and moves it into `CometNativeShuffleSuite`, to reduce the test time.

Reply:
I think by purpose these tests focus on different aspects. For example, "different data type" tests are for shuffling on different data types especially.

They might be some overlapping when we look them now. At beginning, we want to have much more test coverage as possible to reduce missing corner cases.

Reply:
Oops you are right. I got confused when looking at the `execEnabled`.  It's interesting that for this test case,  the check `checkCometExchange(shuffled, 1, true)` can pass regardless of whether `execEnabled` is true or false. Looks like native shuffle can be enabled even if native execution is not.

Reply:
Oh I see. I'll update and keep the `execEnabled`